pkg/controllers/management/node/controller.go

```diff
@@ -254,11 +255,30 @@ func (m *Lifecycle) deployAgent(nodeDir string, obj *v3.Node) error {
 	if err != nil {
 		return err
 	}
+	prefix := "/"
+	cluster, err := m.clusterLister.Get("", obj.Namespace)
+	if err != nil && !kerror.IsNotFound(err) {
+		return err
+	}
```

Comment:
if err is not found, then the cluster will be nil? We should just return nil if not found

Reply:
don' t think if cluster can't be found cause it must have one. If not we just default pathFix to "/"